hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -108,45 +108,45 @@ public static class HiveSyncConfigParams {
             + "instead of org.apache.hudi package. Use this when you are in the process of migrating from "
             + "com.uber.hoodie to org.apache.hudi. Stop using this after you migrated the table definition to "
             + "org.apache.hudi input format.")
-    public Boolean usePreApacheInputFormat;
+    public boolean usePreApacheInputFormat;
```

Comment:
this will make the member variables default to `false` which may differ from the config property 's default value.

Reply:
Sorry for my careless, I added default value to those default value is true.